### DISCUSSION TITLE
Add ability to disable kiai flashes for eye sensitivity concerns

### DISCUSSION THREAD

**Discussion by dawnpace** [ppy/osu#14464] [Ideas] [answered]
Users' eyes are very sensitive to flashes and have difficulty tolerating KiAi's visual effects, and can be distracted by KiAi's flashes, which can lead to performance errors.
Please make a switch to turn it off.
Of course it would be nice to have it in the stb version.

**Comment by yamplum** [comment]
I struggle with kiai flashes a lot. At high star ratings there is a lot of visual processing happening in tight timeframes and kiai completely overloads my vision. Detecting brightness changes in the peripheral vision is the primary mechanism of sightreading, so it's crucial that note targets stay as stable as possible. It's hard to focus on the next note in line when the whole playfield strobes like a nightclub, especially considering it kicks in at high-intensity sections.

Please add an option to turn it off.

**Comment by MaxKruse** [comment]
I would like to add that lazer's kiai glow doesnt match stable and therefore causes a lot of visual strain.

**Reply by frenzibyte** [reply]
That's better tracked in https://github.com/ppy/osu/issues/21747.

**Comment by peppy** [comment, marked as answer]
Tracked via #21747.

**Comment by ghost** [comment]
Please reconsider allowing this to be completely disabled in both stable and lazer, I just can't understand why hit lightning etc. would be toggleable, you can customize or even remove hit100s, hit50s, but kiai flashes need to remain unchangeable. Why on earth is that the hill to die on?
